recipes/libssh/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import collect_libs, copy, get, rmdir
+
+
+class LibSSHRecipe(ConanFile):
+    name = "libssh"
+    version = "0.10.4"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libssh.org/"
+    description = "multiplatform C library implementing the SSHv2 protocol on client and server side"
+    topics = ("libssh", "ssh", "shell", "ssh2", "connection")
```

Comment:
```suggestion
    topics = ("ssh", "shell", "ssh2", "connection")
```
Project's name and `conan` are not good topics